src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -485,7 +485,8 @@ private String generateConferenceId()
             {
                 id
                     = jicofoShortId +
-                        Integer.toHexString(RANDOM.nextInt(0x1_0000));
+                        String.format(
+                            "%4h", RANDOM.nextInt(0x1_0000)).replace(' ', '0');
```

Comment:
what was the issue here?

Reply:
The conference ID consists of one byte for an ID of the jicofo instance plus two bytes for the conference within jicofo. If the random number was < 0x1000 we'd get an ID like "ff123" (with jicofo ID "ff"). This gets serialized the same as 0ff123 which could clash with jicofo ID "0f" and conf ID "f123".

This change pads the random ID so for the case above we get "ff0123".

Reply:
it looks like `String.format("%04x", ...)` does what you need and i think is cleaner than the replace:

```
String.format("%04x", 5); // --> 0005
String.format("%04x", 0xFFFF); // --> ffff
```